src/hooks/usePagination.js

```diff
@@ -1,14 +1,35 @@
 export const DOTS = "...";
 
-function usePagination() {
-  /*
-    Rewrite the logic here to map out the pagination to be displayed
-
-    !!!!!! ATTENTION !!!!!!
-    Please replace this comment here with a description of this hook.
-    
-  */
-  return [1, 2, 3, DOTS, 5];
+function usePagination(object) {
```

Comment:
The value returned by this hook can be memoized/cached to avoid unnecessarily running the logic every time the hook is called. The useMemo hook seems suitable.